packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1398,7 +1398,16 @@ async function handler(
     },
   ];
 
+  const blacklistedGuestEmails = process.env.BLACKLISTED_GUEST_EMAILS
+    ? process.env.BLACKLISTED_GUEST_EMAILS.split(",")
+    : [];
+
+  const guestsRemoved: string[] = [];
   const guests = (reqGuests || []).reduce((guestArray, guest) => {
+    if (blacklistedGuestEmails.some((e) => e === guest)) {
+      guestsRemoved.push(guest);
```

Comment:
We should handle variations of the same email address e.g `email+something` and specifically for Gmail we might need to think about ignoring dots.(Detectable as @gmail.com but work emails give no clue about it)

![image](https://github.com/calcom/cal.com/assets/1780212/28b6a7a9-d2e7-47a8-984d-449750171ef5)

Reply:
Yeah agreed. This PR was a very quick solution just to get a stop-gap in place. I think we can build more on top as we go.